.travis.yml

```diff
@@ -20,6 +20,10 @@ jobs:
     #   script: sbt mima
     - stage: release
       script: sbt ci-release
+    # coverage
+    - env: TEST="Coverage" CI_COVERAGE="true"
```

Comment:
This needs to appear above "stage: release" to run as part of tests, now it will run as release.